### ISSUE TITLE
Main Menu sub-menu dependent on IE11 browser images/pictures turned on

### ISSUE THREAD

**Issue by dorina1** [wet-boew/wet-boew#7544]
For our screen reader users we recommend pictures to be turned off However we discovered that the sub-menu does not work unless the pictures are turned on (IE Options --> Advanced --> Multimedia --> Show Pictures - OFF). Why is the sub-menu dependent on "show pictures - ON"?
many thanks.
@wet-boew/access-working-group

**Comment by dfarough** [comment]
I Just turned pictures off.  this seems to totally disable the menus.  I am not sure why you would want to do this.  It certainly improves performance, but I suspect that you lose a lot.  It seems to me that the user would achieve the same benefits by using the basic html view option at the top of pages.

**Comment by pjackson28** [comment]
@dorina1 That option seems to prevent anything in WET from loading (including all plugins, polyfills and even the Basic HTML view button) which is why the menu is not working. Not sure why that option would have such a major impact unless it affects more than it describes (since many of the affected components don't use images or even CSS background images). Hopefully somebody can figure out what that option in IE actually affects and to figure out why it is having such a profound impact on components that don't use images at all.

**Comment by hsrudnicki** [comment]
Does this remain an issue or otherwise? Please update the status of this again, thank you

**Comment by juleskuehn** [comment]
I can confirm that this remains an issue in IE11 and Chrome 68 (and probably all browsers)
